src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Dict, List
+
+
+def load_from_file(file_path: str) -> Dict[str, List[str]]:
+    """
+    This function extracts labels from a Python file.
+    Labels must be in the first line of a Python file and start with "# labels: "
+    Each label must have the format "key::value1,value2,..."
+
+    Example:
+        "# labels: author:google test_group:daily,monthly"
```

Comment:
Your example doesn't match the rule above.

Rule: 
> Each label must have the format "key::value1,value2,..."

Example:
> author:google test_group:daily,monthly

You're missing a `:`

Reply:
Great catch! Fixed that!